anaconda_anon_usage/tokens.py

```diff
@@ -213,4 +259,7 @@ def token_string(prefix=None, enabled=True):
 
 
 if __name__ == "__main__":
-    print(token_string())
```

Comment:
Just a simple hack I added to make it easy to generate random tokens for testing purposes